aiogram/api/types/user.py

```diff
@@ -33,6 +41,80 @@ class User(TelegramObject):
 
     @property
     def full_name(self) -> str:
+        """
+        Get full name of user.
+        :return: str
```

Comment:
Does it really require doc string? I mean, you'd probably call this method this way: `User.full_name`, i think it's already well documented in terms of API. Also, return type already specified in the annotation, so..